feconf.py

```diff
@@ -295,6 +295,12 @@
 
 
 def get_empty_ratings():
+    """Returns the Empty Ratings.
+
+    Return:
+        JSON: Copy of the '_EMPTY_RATINGS' JSON object which contains the empty
```

Comment:
This isn't the correct typeinfo. The typeinfo should be what you get when you call type() on the return value in Python.

Reply:
Also, can you change `Return:` --> `Returns:` for consistency? As stated above, you can use `type()` to get the type info of the variable and update the docstrings accordingly.

Reply:
Okay, thanks @seanlip  @vibhor98